src/widgets/peoplePicker.js

```diff
@@ -43,29 +43,29 @@ export class PeoplePicker {
       new Group(selectedGroup, this.selectedgroup).render()
       const changeGroupButton = document.createElement('button')
```

Comment:
it would be great to delete this file as it has been corrected in another branch and was a mistake.  I see there is a delete button, but want to confirm with you before using that feature.